src/components/MapView.js

```diff
@@ -261,18 +261,40 @@ class MapView extends Component {
                       onMoveEnd={e => this.onMoveEnd(e)}
                 >
                     {this.renderPopup()}
-                    {reports ? reports.filter(report => dataMatchesFilter(report, filter))
-                        .map(report => (
-                            <Layer type="circle"
-                                key ={report.id}
-                                paint={{'circle-color': getColorForSpecies(report.data.species.toLowerCase()),
-                                    'circle-radius': 7,
-                                    'circle-stroke-width': .3,
-                                    'circle-stroke-opacity': 1}}>
+                    <Layer type="circle"
+                           paint = {{
+                               'circle-radius': 7,
+                               'circle-stroke-width': .3,
+                               'circle-stroke-opacity': 1,
+                               'circle-color': [
+                                   'match',
+                                   ['get', 'species'],
+                                   'Red Fox',
```

Comment:
If these values/colors are used elsewhere in the app, I would suggest moving this to a shared location